src/common/models/series/series.ts

```diff
@@ -30,6 +30,7 @@ export interface BasicSeriesValue {
 export interface SeriesBehaviours {
   key: () => string;
   plywoodKey: (period?: SeriesDerivation) => string;
+  measures: () => string[];
```

Comment:
I can't see tests for measures() implementation. Document the contract/intention for the method or write tests (preferred way).